src/hotspot/os/bsd/os_perf_bsd.cpp

```diff
@@ -26,7 +26,7 @@
 #include "memory/resourceArea.hpp"
 #include "runtime/os.hpp"
 #include "runtime/os_perf.hpp"
-#include "vm_version_ext_x86.hpp"
+//#include "vm_version_ext_x86.hpp"
 
```

Comment:
Looking at the linux version of this file, I believe the correct header include here is:
```
#include CPU_HEADER(vm_version_ext)
```

Reply:
Thanks.  This is better than having this all #ifdef'ed per architecture.